marketing/copy.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Marketing Copy
+description: Text to use on sites which promote Mathesar
+published: true
+date: 2023-02-23T00:00:00.000Z
+tags: 
+editor: markdown
+dateCreated: 2023-02-23T00:00:00.000Z
+---
+
+## Tagline
+
+### Technical
+
+> Mathesar is an open source web application providing an intuitive spreadsheet-like interface to PostgreSQL databases.
```

Comment:
@seancolsen I'm not a fan of "spreadsheet-like" as a descriptor for Mathesar for a few reasons:
- It completely ignores the data explorer, which I think is one of our best features and one of our biggest differentiators from competitors. Our tagline should describe that we support both data entry _and_ data analysis.
- We lack a lot of common features that spreadsheets have (because we're not actually focusing on making Mathesar like a spreadsheet), and I think people may be disappointed.

Reply:
I'm actually not sure if "spreadsheet-like" is my issue here. I think we need more personality to our tagline. This is very dry. I like our Twitter / README ones better:

> An intuitive UI for managing data, for users of all technical skill levels. Built on Postgres.

> Mathesar is a straightforward open source tool that provides a spreadsheet-like interface to a PostgreSQL database.